DP02_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<ul><li>Explain why the it's expected to have the cross shear to be zero (spherical symettry). You could point to ection 3.2 of the Pratt review I mentioned aobve. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Adjusted. Actually it does not have be spherical (more universal). (Gauss's law)</p>